templates/campaigns.html

```diff
@@ -83,6 +83,10 @@ <h4 class="modal-title" id="campaignModalLabel">New Campaign</h4>
               <input type="text" class="form-control" placeholder="http://192.168.1.1" id="url"/>
 	      <label class="control-label" for="url">Schedule: </label>
               <input type="text" class="form-control" id="launch_date"/>
+          <label class="control-label" for="senddelay">Min delay time: <i class="fa fa-question-circle" data-toggle="tooltip" data-placement="right" title="Minimum randomized delay time between sending of each campaign email"></i></label>
```

Comment:
I'd like the delay to be optional. We can have a checkbox that says something like "Randomize Sending Delay" or something that sounds better, heh.

There is code elsewhere (on mobile or I'd link it) that shows how to handle showing options only after a checkbox is shown.